packages/@react-aria/overlays/src/useModal.tsx

```diff
@@ -12,6 +12,7 @@
 
 import React, {AriaAttributes, HTMLAttributes, ReactNode, useContext, useEffect, useMemo, useState} from 'react';
 import ReactDOM from 'react-dom';
+import {useIsSSR} from '../../../react-aria';
```

Comment:
```suggestion
import {useIsSSR} from '@react-aria/ssr';
```

might also need to be added as a dependency.

Reply:
https://github.com/adobe/react-spectrum/pull/3186/commits/be93533b6d84316629d90e9b0f4405d3d8344826 Add a `@react-aria/ssr` dependency in package.json